web/src/App.vue

```diff
@@ -112,4 +129,11 @@ body {
   overflow: auto;
   font-family: 'Roboto';
 }
+
+.v-table__wrapper {
```

Comment:
This fixes the table scrolling issue we had, right?

Reply:
Yes I will add a comment so thats more clear for future maintenance